docs/introduction/open-source/public_code.md

```diff
@@ -1,89 +1,93 @@
+All source code for any policy in use (unless used for fraud detection) MUST be published and publicly accessible. | yes | [code](https://github.com/open-zaak/open-zaak), VNG/GEMMA2 policy linked in [README](https://github.com/open-zaak/open-zaak/blob/master/README.en.md)
+All source code for any software in use (unless used for fraud detection) MUST be published and publicly accessible. | yes | [code](https://github.com/open-zaak/open-zaak)
 Contributors MUST NOT upload sensitive information regarding users, their organization or third parties to the repository. | yes | 2020-05-12 review by @EricHerman; [ISO 27001](https://www.maykinmedia.nl/en/)
 Any source code not currently in use (such as new versions, proposals or older versions) SHOULD be published. | yes | [releases](https://github.com/open-zaak/open-zaak/releases), [docker hub tags](https://hub.docker.com/r/openzaak/open-zaak/tags)
-The source code MAY provide the general public with insight into which source code or policy underpins any specific interaction they have with an organization. | |
-
+Documenting which source code or policy underpins any specific interaction the general public may have with an organization is OPTIONAL. |  |
 
 ## [Bundle policy and source code](https://standard.publiccode.net/criteria/bundle-policy-and-code.html)
 
-- [x] Policy bundled, tested, and linked in the documentation, some policy not available in English
+- [x] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
 A codebase MUST include the policy that the source code is based on. | yes | [API specs embeded in the codebase](https://github.com/open-zaak/open-zaak/blob/master/src/openzaak/components/zaken/openapi.yaml), and linked from [documentation](https://github.com/open-zaak/open-zaak)
-A codebase MUST include all source code that the policy is based on. | | "yes" or "not applicable", do we consider it based on the API?
-All policy and source code that the codebase is based on MUST be documented, reusable and portable. | yes | code dependencies are OSI
+A codebase MUST include all source code that the policy is based on, unless used for fraud detection. |  | "yes" or "not applicable", do we consider it based on the API?
 Policy SHOULD be provided in machine readable and unambiguous formats. | yes | OpenAPI is in machine readable yaml format
 Continuous integration tests SHOULD validate that the source code and the policy are executed coherently. | yes | [GitHub workflow](https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/ci.yml), [API Test Platform](https://api-test.nl/server/1/6b5fe675-694d-4948-8896-5eae88d30ef0/14bc91f7-7d8b-4bba-a020-a6c316655e65/latest/)
 
 ## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
 
-- [x] designed to be reusable from the start
+- [ ] criterion met.
```

Comment:
why is this switched from met to unmet?

Reply:
As discussed in the call, this changes because of the new "codebase SHOULD be localizable" requirement.